python/pyspark/sql/pandas/functions.py

```diff
@@ -0,0 +1,539 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import functools
+import sys
+
+from pyspark import since
+from pyspark.rdd import PythonEvalType
+from pyspark.sql.types import DataType
+from pyspark.sql.udf import _create_udf
+
+
+class PandasUDFType(object):
+    """Pandas UDF Types. See :meth:`pyspark.sql.functions.pandas_udf`.
```

Comment:
sql.pandas.functions.pandas_udf?

Reply:
It still keeps the import as is. I intend to keep `sql.pandas` as rather a private package. I documented at `pandas/__init__.py`.

I double checked this still links currently to the `pandas_udf` in API documentation.

Reply:
Oh I see. Thanks!